service/src/main/kotlin/fi/espoo/evaka/EvakaEnv.kt

```diff
@@ -362,6 +362,22 @@ data class ScheduledJobsEnv(val jobs: Map<ScheduledJob, ScheduledJobSettings>) {
     }
 }
 
+data class OphEnv(
+    val ophOrganizerOid: String,
```

Comment:
Nit: I think the `oph` prefix in the fields is now unnecessary since the fields are grouped in a type that provides some context.

In other words:

```kotlin
data class Person(
  val firstName: String
  val lastName: String
)
```

vs

```kotlin
data class Person(
  val personFirstName: String
  val personLastName: String
)
```